qdrant_client/qdrant_remote.py

```
@@ -2645,3 +2663,55 @@ def collection_cluster_info(self, collection_name: str) -> types.CollectionClust
         ).result
         assert collection_info is not None, "Collection cluster info returned None"
         return collection_info
+
+    def get_optimizations(
+        self,
+        collection_name: str,
+        completed_limit: int | None = None,
+        **kwargs: Any,
```

Ruff (0.15.1)

[warning] Unused method argument: `kwargs`

(ARG002)